common/compositor/gl/glprogram.cpp

```diff
@@ -126,8 +126,116 @@ static std::string GenerateFragmentShader(int layer_count) {
   return fragment_shader_stream.str();
 }
 
```

Comment:
Does entire LoadPrebuiltBinary need to be encapsulated with "ENABLE_PREBUILT_SHADER_BIN_ARRAY "?

Reply:
hmm I thought I replied back to this comment but it seems it was lost..
so we can't do it because LoadPrebuiltBinary is needed for second option as well where it loads shader programs from files.